package.json

```diff
@@ -9,13 +9,12 @@
         "start:dev": "nodemon --exec babel-node server/index.js",
         "heroku-postbuild": "npm run build",
         "test": "nyc mocha --timeout 10000 -r esm --exit",
-        "generate-lcov": "nyc report --reporter=text-lcov > lcov.info",
         "db:rollback": "npm run db:back && npm run db:roll && npm run db:seed",
         "db:roll": "sequelize db:migrate",
         "db:back": "sequelize db:migrate:undo:all",
         "db:seed": "sequelize db:seed:all",
-        "coverage": "npm run generate-lcov"
-    },
+        "coverage": "nyc report --reporter=text-lcov | coveralls",
+},
```

Comment:
The closing bracket is wrongly positioned

Reply:
the script to generate the lcov file will be used by coverage and codeclimate